GVRf/Framework/framework/src/main/jni/vulkan/vulkanCore.h

```diff
@@ -195,6 +206,7 @@ class VulkanCore {
     VkSurfaceKHR m_surface;
 
     std::vector<VkCommandBuffer*> swapChainCmdBuffer;
+    std::vector<VkCommandBuffer*> postEffectCmdBuffer;
 
```

Comment:
It seems you need to have a Vulkan variant of the RenderPass class. Instead of overriding all the functions that create a render pass to make the Vulkan variant, why not define Renderer::createRenderPass and call that everywhere you need to make a render pass?